testbed/Makefile

```diff
@@ -20,6 +20,10 @@ test:
 runtests: test
 	./runtests.sh
 
+.PHONY: listtests
```

Comment:
list testbed?

Reply:
it says "runtests" as the target above, and this is always called with -C testbed from parent makefile so...